pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -103,8 +103,11 @@ func (k *Kubernetes) CheckUnsupportedKey(komposeObject *kobject.KomposeObject, u
 }
 
 // InitPodSpec creates the pod specification
-func (k *Kubernetes) InitPodSpec(name string, image string) api.PodSpec {
+func (k *Kubernetes) InitPodSpec(name string, image string, groupadd []int64) api.PodSpec {
```

Comment:
this is bad, no need to add groupadd here, we can add this to k8sutils.go, having it in kubernetes.go / openshift.go will make initpodspec messy.